tutorial.md

```diff
@@ -170,5 +195,14 @@ git remote set-url origin [git-repo-url]
 ```
 
 Review the modified files, commit and push to your remote repository branch.
+
+## Delete the deployed solution
+
+Optional: Use one of the below options in case you want to delete the deployed solution
+
+* Go to [JSS deployment page](https://console.cloud.google.com/products/solutions/deployments?pageState=(%22deployments%22:(%22f%22:%22%255B%257B_22k_22_3A_22Labels_22_2C_22t_22_3A13_2C_22v_22_3A_22_5C_22modification-reason%2520_3A%2520make-it-mine_5C_22_22_2C_22s_22_3Atrue_2C_22i_22_3A_22deployment.labels_22%257D%255D%22))).
```

Comment:
I think the page is called Solution deployments

Reply:
Sure. Done.